tensorflow/lite/experimental/acceleration/configuration/configuration.proto

```diff
@@ -622,6 +624,14 @@ message CPUSettings {
   optional int32 num_threads = 1 [default = -1];
 }
 
+// Arm NN Delegate Settings
+// More information about Arm NN delegate options can be found in https://arm-software.github.io/armnn/latest/delegate.xhtml
+message ArmNNSettings {
+  optional string backends = 1;
+  optional bool fastmath = 2;
+  optional string additional_parameters = 3;
```

Comment:
Thanks for raising this CL!

I understand that you added a link to the Arm NN delegate options on line 628.  I think it would nevertheless be useful to have a brief description of the fields and their meaning in this file too.  That's also what we have for the other delegate settings in this file.

Would you consider adding this?